datumaro/requirements.txt

```diff
@@ -8,3 +8,4 @@ pycocotools>=2.0.0
 PyYAML>=5.1.1
 scikit-image>=0.15.0
 tensorboardX>=1.8
+defusedxml>=0.6.0
```

Comment:
It'd be better to maintain alphabetical order, unless some magic required (like with Cython).